high_card.rb

```diff
@@ -1,5 +1,58 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
+require "pry"
+@ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+@suits = [ "hearts", "spades", "clubs", "diamonds" ]
+@deck = []
+@shuffled_deck = []
+@players = []
+@cards = []
+@sorted_cards = []
 
-players = []
+attr_accessor = :winner
```

Comment:
You don't need to use `attr_accessor` here since we are not defining a Ruby class